incident_response_checklist.md

```diff
@@ -6,25 +6,27 @@ For on-call engineers
 - <input type='checkbox'> **From the PagerDuty incident, click *More Actions* -> *Create JIRA Issue* to create a ticket on the [Production Board](https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=15&projectKey=PROD&selectedIssue=PROD-324)**
     - This will create a Jira ticket that includes some metadata about the incident, including start-time.  Update the summary and description as needed.
     - Assign to yourself if you are investigating the issue
-- <input type='checkbox'>**Report in the #workbench-oncall slack channel that you are investigating event**
+- <input type='checkbox'> **Report in the #workbench-oncall slack channel that you are investigating event**
     - If it impacts many users:
         - Daytime: Confirm that a support specialist is aware of the issue (slack #dsde-comms or @dspsupportteam in #workbench-resilience) and can put up a FireCloud/Terra [Service Incident](https://broadinstitute.zendesk.com/hc/en-us/sections/360003692231-Service-Notifications) and banner. Provide the Jira ticket and a description of the user impact.
         - Off-hours: [Follow this communication playbook to put up an initial banner.](https://docs.google.com/document/d/1E2qSIQECBBS0daWa_VXAOprdV5H_zvirgryTbxbPTDg/edit)
     - Impacts a single user (the user will need to be directly contacted)
         - Daytime: contact a support specialist for assistance (slack #dsde-comms or @dspsupportteam in #workbench-resilience). 
         - Off-hours: Contact the user and cc support@terra.bio . The email should explain the time the issue occurred, how the issue is impacting the user, and any relevant Terra details (workspace/submission/notebook/etc.) The Frontline support team will follow up with the user the next work day to answer any further questions.
-- <input type='checkbox'> **Oversee the remediation of the issue, returning the affected systems to normal.**  Follow steps in the General Troubleshooting Playbook.
+- <input type='checkbox'> **When user communication has occurred, update the "Users Informed" field on the Jira bug.** 
```

Comment:
is this for support team to do or dev?

Reply:
Dev to do if Frontline hasn't. on-calls may be handling issues with no frontline involved but this still needs to be done.  i'll make that clearer